server_addon/applications/server/settings.py

```diff
@@ -144,7 +144,17 @@ def validate_unique_name(cls, value):
 
 
 class ApplicationsSettings(BaseSettingsModel):
-    """Applications settings"""
+    """Applications settings
+
+    Define applications are available to your studio, locations of their
+    executables, and their additional environments.
+
+    AYON uses the following definitions to populate the applications list
+    in project anatomy > attributes.
+    So, You'd still need modify your project anatomy settings.
```

Comment:
This comment is unclear to me. What are you trying to explain here?

Reply:
Ah, that makes sense. I have a hard time describing that without a perspective of what the anatomy settings are. 

```suggestion
    AYON uses the Applications Settings definition to build the
    list of all available applications in the studio settings list. 
    Those then define the application variants that can be 
    enabled per project in the project anatomy settings.
```

Not sure if that's any better. I feel like there must be a better way to describe this with not too many words, but I can't think of any.

Reply:
what about explaining the end result ?
```suggestion
Hint: 
    AYON Launcher will show the following applications list filtered by your project anatomy settings.
```

Reply:
I personally like that wording less - but maybe someone else can chime in.

Reply:
tagging @dee-ynput for information.